pyfdb/pyfdb.py

```diff
@@ -275,18 +290,47 @@ class FDB:
 
     __fdb = None
 
-    def __init__(self):
+    def __init__(self, config=None, user_config=None):
         fdb = ffi.new("fdb_handle_t**")
-        lib.fdb_new_handle(fdb)
+
+        if config is not None or user_config is not None:
```

Comment:
No. We should update the required version of the FDB, so that the interfaces match. Butter than having this ad-hoc testing of interfaces.